llvm/docs/LangRef.rst

```diff
@@ -3572,6 +3572,29 @@ or ``syncscope("<target-scope>")`` *synchronizes with* and participates in the
 seq\_cst total orderings of other operations that are not marked
 ``syncscope("singlethread")`` or ``syncscope("<target-scope>")``.
 
+.. _floatsem:
+
+Floating-Point Semantics
+------------------------
+
+LLVM floating-point types fall into two categories:
+
+- half, float, double, and fp128, which correspond to the binary16, binary32,
+  binary64, and binary128 formats described in the IEEE-754 specification.
```

Comment:
I think it would be useful to modify the Floating-point type section so that (e.g.) `half` is specifically described as "IEEE 754 binary16", in addition to the changes here.

Reply:
Currently there is this sentence below that section:
"The binary format of half, float, double, and fp128 correspond to the IEEE-754-2008 specifications for binary16, binary32, binary64, and binary128 respectively."
So you are suggesting that should be moved inside the table? Sure, I can do that.